### PR TITLE
Move namespace from Delta to Deltas

### LINKED ISSUE #51: rename "Delta" namespace to "Deltas"
It is more appropriate and will also allow cleaning all the usage of the class from 
`Protocol.Delta.Delta` to `Delta`

### PR DESCRIPTION
**Pull request by monsieurleberre**
closes #51 
1) Delta as a namespace is renamed to Deltas
~2) use TimeStamp from protobuff known types to represent the timestamp in the delta~
done in a separate PR https://github.com/catalyst-network/protocol-protobuffs/pull/54

### CODE REVIEW

File: src/Deltas.proto [hunk 1]
```diff
@@ -43,7 +44,7 @@ package Catalyst.Protocol.Delta;
 	bytes PreviousDeltaDfsHash = 2;
 	bytes MerkleRoot = 3;
 	bytes MerklePoda = 4;
-	uint32 TimeStamp = 5;
+	google.protobuf.Timestamp TimeStamp = 5;
```
Comment (nshCore) [inline src/Deltas.proto]: could this not be in a separate PR as timestamp changes aren't related to namespace naming convention changes?

**Review by nshCore — CHANGES_REQUESTED** [review]

File: src/Deltas.proto (hunk 1 above)
Comment (monsieurleberre) [reply src/Deltas.proto]: 😂 sorry for the cognitive overload, I overestimated you 😂 this PR is doing what it says in the description though ;)

Comment (monsieurleberre) [reply src/Deltas.proto]: https://github.com/catalyst-network/protocol-protobuffs/pull/54

**Review by Millymanz — DISMISSED** [review]

**Review by nshCore — DISMISSED** [review]
